extensions/functions_string.yaml

```diff
@@ -17,6 +17,27 @@ scalar_functions:
         variadic:
           min: 1
         return: "string"
+  - name: chr
+    description: >-
+    Returns the ASCII character having the binary equivalent to input.
+    If n is larger than 256 the result is equivalent to chr(n % 256)
```

Comment:
what's the behavior for negative values of n?

Reply:
Add description for negative values.